{"filter":false,"title":"my_package_launch_file.launch","tooltip":"/my_package/launch/my_package_launch_file.launch","undoManager":{"mark":1,"position":1,"stack":[[{"start":{"row":0,"column":0},"end":{"row":7,"column":9},"action":"insert","lines":["<launch>","  <!-- turtlebot_teleop_key already has its own built in velocity smoother -->","  <node pkg=\"turtlebot_teleop\" type=\"turtlebot_teleop_key.py\" name=\"turtlebot_teleop_keyboard\"  output=\"screen\">","    <param name=\"scale_linear\" value=\"0.5\" type=\"double\"/>","    <param name=\"scale_angular\" value=\"1.5\" type=\"double\"/>","    <remap from=\"turtlebot_teleop_keyboard/cmd_vel\" to=\"/cmd_vel\"/>   <!-- cmd_vel_mux/input/teleop\"/-->","  </node>","</launch>"],"id":1}],[{"start":{"row":0,"column":0},"end":{"row":7,"column":9},"action":"remove","lines":["<launch>","  <!-- turtlebot_teleop_key already has its own built in velocity smoother -->","  <node pkg=\"turtlebot_teleop\" type=\"turtlebot_teleop_key.py\" name=\"turtlebot_teleop_keyboard\"  output=\"screen\">","    <param name=\"scale_linear\" value=\"0.5\" type=\"double\"/>","    <param name=\"scale_angular\" value=\"1.5\" type=\"double\"/>","    <remap from=\"turtlebot_teleop_keyboard/cmd_vel\" to=\"/cmd_vel\"/>   <!-- cmd_vel_mux/input/teleop\"/-->","  </node>","</launch>"],"id":2},{"start":{"row":0,"column":0},"end":{"row":4,"column":9},"action":"insert","lines":["<launch>","    <!-- My Package launch file -->","    <node pkg=\"my_package\" type=\"simple.py\" name=\"ObiWan\"  output=\"screen\">","    </node>","</launch>"]}]]},"ace":{"folds":[],"scrolltop":0,"scrollleft":0,"selection":{"start":{"row":0,"column":0},"end":{"row":4,"column":9},"isBackwards":false},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"hash":"0d8dc671120f26b254dd110cb6f3482d5e298d9a","timestamp":1538450974000}